freebsd-kvm/base-image/http/installerconfig

```diff
@@ -0,0 +1,17 @@
+DISTRIBUTIONS="kernel.txz base.txz"
+export ZFSBOOT_DISKS="vtbd0"
+export nonInteractive="YES"
+
+#!/bin/sh
+
+sysrc ifconfig_DEFAULT=SYNCDHCP
+sysrc sshd_enable=YES
+sysrc sendmail_enable=NONE
+sysrc -f /boot/loader.conf autoboot_delay=-1
+
+sed -i '' -e 's/^#UseDNS yes/UseDNS no/' /etc/ssh/sshd_config
+sed -i '' -e 's/^#PermitRootLogin no/PermitRootLogin yes/' /etc/ssh/sshd_config
```

Comment:
Why `UseDNS no`?  Just curious.

Reply:
My knowledge of these things is not particularly deep so I could well be misunderstanding, but I recall reading (not specifically in the context of FreeBSD) that disabling DNS lookups can improve how quickly ssh is able to connect and if DNS is not functioning properly, `UseDNS yes` can apparently keep you from being able to log in at all. If that is false or it would be otherwise valuable to keep it, I'm happy to remove this!